.codespell_dict

```diff
@@ -0,0 +1,3 @@
+sub-directory->subdirectory
```

Comment:
Did you mean to drop sub-directories?

```suggestion
sub-directory->subdirectory
sub-directories->subdirectories
```

Reply:
yes, because plurals and capitalization are automatically picked up, I think

Reply:
tested in https://github.com/bids-standard/bids-specification/pull/1322/commits/2bc1b567a8256df2a0ca4ffc275a97d7e2c0cea4, and CI passed. So apparently plurals are not necessarily caught automatically, I added them back to the dict in https://github.com/bids-standard/bids-specification/pull/1322/commits/9e1ddd2f48432aac14217a718d18252c2eea6d7f